frontend/packages/dev-console/integration-tests/features/guided-tour/quick-starts/quick_start_card_on_cluster_overview_ODC-4922.feature

```diff
@@ -0,0 +1,53 @@
+Feature: Add quick starts card on cluster overview
+	As a user, I want to easily be able to access and discover Quick Starts from the Cluster Overview in administrator view.
+
+Background:
+    Given user is at administrator perspective
+    And user has created cr for sample-application-quickstart quick start
+    And user has created cr for install-associate-pipeline-quickstart quick start
+    And user has created cr for explore-pipeline-quickstart quick start
+    And user has created cr for add-healthchecks-quickstart quick start
+
+@regression
+Scenario: Quick Start card on Cluster Overview 
+   Given user is in administrator perspective
+   When user goes to Cluster Overview page
+   Then user can see quick tour card 
+   And user can see three Quick Start link present on it
+   And user can see the "See all Quick Starts" 
+   And user can see the kebab menu on top right of the card
+
+@regression
+Scenario: Quick start links with status as in progress 
+   Given user is at Cluster Overview page
+   When user clicks on first quick start link on the quick start card 
+   And user clicks on the Start tour
+   And user clicks on close button
+   And user clicks on leave on leave the tour modal box
+   Then user can see "In Progress" status below the first quick start link
```

Comment:
I didn't see any pop up as per your statements. Also I didn't see any In Progress status on card

Reply:
You should add few more steps.

Reply:
I can see both leave the tour modal box as well as "In Progress" status

Reply:
Try on another build